### PR TITLE
Add support for inlineStyles in options argument

### PR DESCRIPTION
**Pull request by sstur**
This is a combination of the features in #12, #15 and #24 rolled into one diff. Based on the work of @adamtal3 in #24.

### CODE REVIEW

File: src/stateToHTML.js [hunk 1]
```diff
@@ -103,6 +142,73 @@ function getWrapperTag(blockType: string): ?string {
...
+    } else {
+      styleMap[styleName] = customMap[styleName];
+      styleOrder.push(styleName);
+    }
+  }
+  return [styleMap, styleOrder];
+}
+
+function styleToCSS(styleDescr: StyleDescr): string {
+  let parts = Object.keys(styleDescr).map((name) => {
```
Comment (coopy) [inline src/stateToHTML.js]: It's getting busy in here :)
These transform functions (`styleToCSS`, `processStyleName|Value`, `normalizeAttributes`) could be extracted to their own modules for ease of reading and testing.

Comment (coopy) [inline src/stateToHTML.js]: Could just `return Object.keys(styleDescr).map(...).join('; ');` without creating the intermediate variable.

**Comment by coopy** [comment]
Nice combo! 😸

**Comment by adamtal3** [comment]
Looks great! This is much more customizable than my suggestion which makes it much better :smile: 

What about https://github.com/sstur/draft-js-import-element/pull/13 ? Do you wish to implement something similar for the html import?

Thanks for your effort!

**Comment by sstur** [comment]
Thanks to @coopy for the testing he did in #15. I took an almost identical approach here.

**Comment by sstur** [comment]
@adamtal3: I do have a plan for https://github.com/sstur/draft-js-import-element/pull/13. I'll continue this conversation over there. Thanks!

**Pull request merged**